classes/log/store.php

```diff
@@ -118,11 +146,18 @@ public function process_events(array $events) {
             $eventbatches = array_chunk($translatorevents, $maxbatchsize);
         }
 
+		$translator_event = new Event();
+		$translator_event_read_return = @$translator_event->read(array());
```

Comment:
in the emitter repository (for which I believe I hadn't had the time to make a PR yet).